lib/rbs_rails/active_record.rb

```diff
@@ -529,11 +529,11 @@ def authenticate_#{attribute}: (::String) -> (#{klass_name} | false)
             def #{col.name}: () -> #{column_type}
             def #{col.name}=: (#{column_type}) -> #{column_type}
             def #{col.name}?: () -> bool
-            def #{col.name}_changed?: () -> bool
+            def #{col.name}_changed?: (?from: #{class_name_opt}, ?to: #{class_name_opt}) -> bool
```

Comment:
Note: The documentation of `#*_changed?` does not describe these options. But the implementation of `#attribute_changed?` takes them ([ref1](https://github.com/rails/rails/blob/v8.1.1/activemodel/lib/active_model/dirty.rb#L300), [ref2](https://github.com/rails/rails/blob/v8.1.1/activemodel/lib/active_model/attribute_mutation_tracker.rb#L44)).